2019/python/rules_engine.py

```diff
@@ -1898,13 +1898,13 @@ def v676_5(self):
 
 	def v677_1(self):
 		"""An invalid Interest Rate was reported.
-		1) Interest Rate must be a number greater than 0 or NA, and cannot be left blank.
+		1) Interest Rate must be a number greater than or equal to 0 or NA, and cannot be left blank.
 
```

Comment:
Please combine these into the language from the FIG. The separation was due to S2155 coming out so late last year.

Reply:
Sounds good!